eng/testing/tests.singlefile.targets

```diff
@@ -103,6 +107,17 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="RemoveDebuggingBinariesFromTestR2R"
+          Condition="'$(TestReadyToRun)' == 'true'"
+          BeforeTargets="__ExcludeAssembliesFromSingleFile">
+    <ItemGroup>
+      <FilesToRemove Include="$(OutDir)*.dbg" />
+      <FilesToRemove Include="$(PublishDir)*.dbg" />
+    </ItemGroup>
+
+    <Delete Files="@(FilesToRemove)" />
+  </Target>
```

Comment:
There must be a cleaner way to do this. We should avoid these files to get copied.